english/cricket/cricket.md

```diff
@@ -3,6 +3,8 @@ Click [here](http://www.espncricinfo.com) to catch up on cricket news and view l
 You look around and find the cricket bat-and-ball game equipment and rule book. 
 Click [here](../cricket-rules/cricket-rule-book.md) to read the rules.
 
+You decide that racquetball is a better sport Click [here](../../racquetball/racquetball.md).
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.